snapchat_dl/snapchat_dl.py

```diff
@@ -90,6 +90,10 @@ def download(self, username):
         Returns:
             [bool]: story downloader
         """
+        stories_already_exists = 0
+        stories_downloaded = 0
```

Comment:
I think we don't need this, we can simply compute it by doing subtraction `stories.length - story_download_count`